recipes-core/u-boot/files/boot.cmd.pvgeneric

```diff
@@ -181,18 +182,20 @@ echo "load ${devtype} ${pv_mmcdev}:${pv_mmcdata} ${fdt_addr_r} /trails/${boot_re
 if load ${devtype} ${pv_mmcdev}:${pv_mmcdata} ${fdt_addr_r} /trails/${boot_rev}/.pv/pv-fdt.dtb; then
 	echo "Successfully loaded pv-fdt.dtb"
 else
+	# sanitize fdt file in case of leading directories (always flat in PV BSP)
+	setexpr fdtfile gsub '.*/' '' ${fdtfile}
```

Comment:
this is actually not the case always .... lets look at the case here. on other platforms we have proper directory inside bsp/ folder after installing like bsp/freescale/blah.dtb.... lets see what is happening here and why that directory is not preserved.

Reply:
In here, I don't know why this was added. Maybe @ricmm knows more about it.

Reply:
ok i see the problem. i will fix it and then lets remove this change.

Reply:
ok pushing with this change removed. now fdtfile allwinner/orangepi-3lts.dtb will resolve correctly inside bsp/ folder of trail.

Reply:
but this breaks OTA so we need to release a new previw image....